src/briefcase/integrations/linuxdeploy.py

```diff
@@ -48,6 +49,21 @@ def download_url(self) -> str:
     def file_path(self) -> Path:
         """The folder on the local filesystem that contains the file_name."""
 
+    @classmethod
+    def arch(cls, host_os: str, host_arch: str) -> str:
+        # always use the x86-64 arch on macOS since Docker
+        # containers are always run in an x86-64 VM
+        arch = host_arch if host_os != "Darwin" else "x86_64"
```

Comment:
Needs confirmation from someone with Apple silicon hardware.

Reply:
As noted in the longer comment - both x86_64 and arm docker images are a possibility.